flaky: test_submit_sleep_with_max_running

failure is on mac: https://github.com/equinor/ert/actions/runs/7758028506/job/21158986284
```python

submit_sleep = 0.01, realization_max_runtime = 0.01, max_running = 10
storage = <ert.storage.local_storage.LocalStorageAccessor object at 0x13aadc4c0>
tmp_path = PosixPath('/private/var/folders/qv/pdh5wsgn0lq3dp77zj602b5c0000gn/T/pytest-of-runner/pytest-0/popen-gw0/test_submit_sleep_with_max_run4')
mock_driver = <class 'conftest.MockDriver'>

    @flaky(max_runs=5, min_passes=1)
    @pytest.mark.parametrize(
        "submit_sleep, realization_max_runtime, max_running",
        [
            (0.01, 0.01, 1),
            (0.01, 0.01, 10),
            (0.01, 0.1, 5),
        ],
    )
    async def test_submit_sleep_with_max_running(
        submit_sleep, realization_max_runtime, max_running, storage, tmp_path, mock_driver
    ):
        run_start_times: List[float] = []
    
        async def wait():
            nonlocal run_start_times
            run_start_times.append(time.time())
            # If the realization runtimes are constant, we will never get into
            # the situation where we can start many realizations at the same moment
            runtime = realization_max_runtime * random.random()
            await asyncio.sleep(runtime)
    
        ensemble_size = 10
    
        ensemble = storage.create_experiment().create_ensemble(
            name="foo", ensemble_size=ensemble_size
        )
        realizations = [
            create_stub_realization(ensemble, tmp_path, iens)
            for iens in range(ensemble_size)
        ]
    
        sch = scheduler.Scheduler(
            mock_driver(wait=wait),
            realizations,
            submit_sleep=submit_sleep,
            max_running=max_running,
        )
        await sch.execute()
    
        deltas = [
            next_start - start
            for start, next_start in zip(run_start_times[:-1], run_start_times[1:])
        ]
>       assert min(deltas) >= submit_sleep * 0.8
E       assert 0.003916025161743164 >= (0.01 * 0.8)
E        +  where 0.003916025161743164 = min([0.011023998260498047, 0.010131120681762695, 0.009545087814331055, 0.009773731231689453, 0.02090907096862793, 0.006201028823852539, ...])
```

--- a/tests/unit_tests/scheduler/test_scheduler.py
+++ b/tests/unit_tests/scheduler/test_scheduler.py
@@ -403,9 +403,9 @@ async def test_submit_sleep(
 @pytest.mark.parametrize(
     "submit_sleep, realization_max_runtime, max_running",
     [
-        (0.01, 0.01, 1),
-        (0.01, 0.01, 10),
-        (0.01, 0.1, 5),
+        (0.15, 0.10, 1),
+        (0.15, 0.10, 10),
+        (0.15, 0.35, 5),
     ],
 )
 async def test_submit_sleep_with_max_running(

